### ISSUE TITLE
d2a58c2_nohealthcheck doesn't build executable because git is missing

### ISSUE THREAD

**Issue by sgofferj** [fredclausen/docker-rtlsdrairband#37]
```
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service libseccomp2: starting
s6-rc: info: service 01-build-rtl_airband: starting
Building rtl_airband optimised for this host's CPU
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service libseccomp2 successfully started
s6-rc: info: service legacy-cont-init successfully started
Building with "armv8-generic" optimisations.
WARNING: Unable to determine architecture, will build with no optimisations.
[building rtlair_band: cmake] CMake Error at CMakeLists.txt:13 (message):
[building rtlair_band: cmake]   Failed to detect RTL_AIRBAND_VERSION -
[building rtlair_band: cmake]   "/opt/rtlsdr-airband/scripts/find_version: line 9: git: command not found"
[building rtlair_band: cmake] -- Configuring incomplete, errors occurred!
[building rtlair_band: cmake] See also "/opt/rtlsdr-airband/build/CMakeFiles/CMakeOutput.log".
[building rtlair_band: make] make: *** No targets specified and no makefile found.  Stop.
[building rtlair_band: make install] make: *** No rule to make target 'install'.  Stop.
s6-rc: info: service 01-build-rtl_airband successfully started
s6-rc: info: service 02-icecast: starting
s6-rc: info: service 02-icecast successfully started
s6-rc: info: service 03-rtl_airband: starting
rtlsdr-airband custom config provided. Ignoring RTLSDRBAND config options
s6-rc: info: service 03-rtl_airband successfully started
s6-rc: info: service rtlsdr-airband: starting
s6-rc: info: service prometheus: starting
s6-rc: info: service icecast: starting
s6-rc: info: service rtlsdr-airband successfully started
s6-rc: info: service prometheus successfully started
s6-rc: info: service icecast successfully started
s6-rc: info: service legacy-services: starting
cat: /tmp/rtl_airband_stats.txt: No such file or directory
[icecast] 2024/02/07 06:23:47 Starting Icecast
[rtlsdr-airband] 2024/02/07 06:23:47 Waiting for local icecast to start...
s6-rc: info: service legacy-services successfully started
[icecast] 2024/02/07 06:23:47 [2024-02-07  06:23:47] WARN CONFIG/_parse_root Warning, <hostname> not configured, using default value "localhost". This will cause problems, e.g. with YP directory listings.
[rtlsdr-airband] 2024/02/07 06:23:49 /etc/s6-overlay/scripts/rtlsdr-airband: line 22: /usr/local/bin/rtl_airband: No such file or directory
cat: /tmp/rtl_airband_stats.txt: No such file or directory
[rtlsdr-airband] 2024/02/07 06:23:50 /etc/s6-overlay/scripts/rtlsdr-airband: line 22: /usr/local/bin/rtl_airband: No such file or directory
[rtlsdr-airband] 2024/02/07 06:23:51 /etc/s6-overlay/scripts/rtlsdr-airband: line 22: /usr/local/bin/rtl_airband: No such file or directory
[rtlsdr-airband] 2024/02/07 06:23:52 /etc/s6-overlay/scripts/rtlsdr-airband: line 22: /usr/local/bin/rtl_airband: No such file or directory
[rtlsdr-airband] 2024/02/07 06:23:53 /etc/s6-overlay/scripts/rtlsdr-airband: line 22: /usr/local/bin/rtl_airband: No such file or directory
```

**Comment by fredclausen** [comment]
This should be fixed in #38 once the new build is done. Please reopen if you experience any more issues.

**Comment by sgofferj** [comment]
e625d19_linux_arm64_nohealthcheck seems to work fine now, thanks.
